include/tins/ip.h

```diff
@@ -379,6 +379,12 @@ class TINS_API IP : public PDU {
         return Endian::be_to_host(header_.check);
     }
 
+    /**
+     * \brief Setter for the checksum field. Disables calculation
+     * during write serialization
+     */
+    void checksum(uint16_t new_check);
+
```

Comment:
Move to the setters section after the `protocol` setter.

Reply:
I missed this one, let me retest and push this change